snippets/unordered_map/bucket.cpp

```diff
@@ -0,0 +1,23 @@
+#include <bits/stdc++.h> 
```

Comment:
Please refrain from using this. It is recommended to use separate header files for specific purpose.